languages/csharp/AppEncryption/AppEncryption.Tests/AppEncryption/Kms/AwsKeyManagementServiceImplTest.cs

```diff
@@ -20,9 +20,7 @@
 using Newtonsoft.Json.Linq;
 using Xunit;
 
-using static GoDaddy.Asherah.AppEncryption.KeyManagement.AwsKeyManagementServiceImpl;
```

Comment:
i think the test code is cleaner if we keep this to avoid all the constant prefixing. it's making some of the lines go beyond 120 in the file now